app/models/patient.rb

```diff
@@ -450,6 +450,18 @@ def assessmenmts_summary_array(assessment_headers, symptoms_headers)
     assessments_summary
   end
 
+  def calc_current_age
+    dob = date_of_birth
+    today = Date.today
+    age = today.year - dob.year
+    age -= 1 if
+        (dob.month > today.month) ||
+        ((dob.month >= today.month) && (dob.day > today.day))
+
+    update(age: age) if self.age != age
```

Comment:
I like this method, but I don't think it should be doing any updating (should just be informational, like for filling out sms message). I think it is fine letting the front end "suggest" an age when manually enrolling, but we don't want to ever overwrite that behind the scenes. I would suggest we remove this line, and instead at the very beginning return age if it exists, else go into this logic.